### PR TITLE
Mtl h hdmi display

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: src/mainboard/clevo/mtl-h/variants/igpu/overridetree.cb [hunk 1]
```diff
@@ -4,7 +4,7 @@ chip soc/intel/meteorlake
 			register "ddi_port_A_config" = "1"
 			register "ddi_ports_config" = "{
 				[DDI_PORT_A] = DDI_ENABLE_HPD, 			/* eDP */
-				[DDI_PORT_2] = DDI_ENABLE_DDC | DDI_ENABLE_HPD,	/* HDMI 2.1 */
+				[DDI_PORT_3] = DDI_ENABLE_DDC | DDI_ENABLE_HPD,	/* HDMI 2.1 */
```
Comment (mkopec) [inline src/mainboard/clevo/mtl-h/variants/igpu/overridetree.cb:7]: oof, thanks for fixing this

File: src/mainboard/clevo/mtl-h/variants/igpu/gpio.c [hunk 2]
```diff
@@ -198,13 +198,13 @@ static const struct pad_config gpio_table[] = {
...
 
-	/* GPP_C11 - SRCCLKREQ2# */
+	/* GPP_C11 - SRCCLKREQ2# (LAN) */
 	/* DW0: 0x44000702, DW1: 0x00000000 */
 	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
 	/* PAD_CFG_NF(GPP_C11, NONE, DEEP, NF1), */
 	_PAD_CFG_STRUCT(GPP_C11, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1), 0),
 
-	/* GPP_C12 - SRCCLKREQ3# */
+	/* GPP_C12 - SRCCLKREQ3# (SDCRAD) */
```
Comment (mkopec) [inline src/mainboard/clevo/mtl-h/variants/igpu/gpio.c:207]:
```suggestion
	/* GPP_C12 - SRCCLKREQ3# (SDCARD) */
```

**Review by mkopec — APPROVED** [review]
Just a small typo.

Curious that you needed to make changes to VBT, considering I dumped it from vendor FW.

**Comment by mkopec** [comment]
@miczyg1 please drop the first 5 commits, I reverted `mtl-h` to a state from before making changes for nvidia

File: src/mainboard/clevo/mtl-h/variants/igpu/gpio.c (hunk 2 above)
Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/variants/igpu/gpio.c]: Fixed: https://github.com/Dasharo/coreboot/pull/547/files#diff-bee32b3061dcb96378bd5e43239dd264fcbea7856d0046648d1df4cb6d61808bR207

**Comment by miczyg1** [comment]
> @miczyg1 please drop the first 5 commits, I reverted `mtl-h` to a state from before making changes for nvidia

Done

**Review by mkopec — APPROVED** [review]
Thanks.

No, I hadn't tested HDMI and TBT video in BIOS on Insyde. I'm not even sure if they set `LidStatus` in GOP correctly.

**Pull request merged**